modular.html

```diff
@@ -0,0 +1,74 @@
+<h2>Creating modular projects</h2>
+
+<p>
+    If your JavaFX 11 project is modular then you need to follow these instructions.
+</p>
+
+<p>
+    Download the appropriate
+    <a target="_blank" href="https://gluonhq.com/products/javafx/">JavaFX jmods</a>
```

Comment:
Is it necessary to use JavaFX JMods? Can't someone use the JavaFX SDK with jars and achieve the same result?

Reply:
Without jmods, jlink doesn't work. Notice the native libraries are bundled within the jmods, but not with the SDK jars.